packages/cashier/src/types/menu-option.types.ts

```diff
@@ -0,0 +1,9 @@
+export type TMenuOption = {
```

Comment:
@vinu-deriv If I'm not mistaken it’s only been used in `packages/cashier/src/pages/on-ramp/on-ramp.tsx`, So we can scope it there and there is no need to expose it to the whole project.